### PR TITLE
Update Readme and AWS Blog example

### PR DESCRIPTION
**Pull request by kowshikn**
*Issue #304* 

*Description of changes:*

* Moved the blog example sh to the branch.
* Added the otel-collector-config with batch processor. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/aws/jaeger-hotrod-on-ec2/setup-jaeger-hotrod.sh [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+cat <<EOT >> otel-collector-config.yml
+receivers:
+  jaeger:
+    protocols:
+      grpc:
+
+processors:
+  batch/traces:
+    timeout: 1s
+    send_batch_size: 50
```
Comment (wrijeff) [inline examples/aws/jaeger-hotrod-on-ec2/setup-jaeger-hotrod.sh:29]: Tested these locally right?

**Review by wrijeff — APPROVED** [review]
👍

File: examples/aws/jaeger-hotrod-on-ec2/setup-jaeger-hotrod.sh (hunk 1 above)
Comment (kowshikn) [reply examples/aws/jaeger-hotrod-on-ec2/setup-jaeger-hotrod.sh:29]: Yes. Added another commit to this PR with readme changes.

**Comment by wrijeff** [comment]
Readme changes are good 👍 . We can revisit docs when we start adding the scaling/tuning guidelines

**Pull request merged**